ArraysHomework/ArraysHomework.cs

```diff
@@ -125,7 +138,9 @@ public void Lists()
             Assert.IsTrue(copiedList[2] == 'c');
 
             // Swap the last and first value of this list.
-            List<string> swapMyValues = new List<string> { "first", "middle", "last" };
+            List<string> swapMyValues = new List<string>() { "first", "middle", "last" };
+            swapMyValues[0] = "last";
```

Comment:
How would you do this if you couldn't see how swapMyValues was initialized (so you didn't know its length or the values of its elements)?